src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemDefault.tsx

```diff
@@ -18,24 +18,55 @@ export const SelectedItemDefault: (props: IPeoplePickerItemProps) => JSX.Element
     <div
       className={css('ms-PickerPersona-container', {
         'is-selected': isSelected
-      }) }
-      data-is-focusable={ true }
-      data-selection-index={ index }
-      key={ index } >
+      })}
+      data-is-focusable={true}
+      data-selection-index={index}
+      key={index} >
       <div className='ms-PickerItem-content'>
         <Persona
           { ...item }
-          presence = { item.presence ? item.presence : PersonaPresence.online }
-          size={ PersonaSize.extraSmall }
+          presence={item.presence ? item.presence : PersonaPresence.online}
+          size={PersonaSize.extraSmall}
           />
       </div>
       <Button
-        onClick={ () => { if (onRemoveItem) { onRemoveItem(); } } }
-        icon={ 'Cancel' }
-        buttonType={ ButtonType.icon }
+        onClick={() => { if (onRemoveItem) { onRemoveItem(); } } }
+        icon={'Cancel'}
+        buttonType={ButtonType.icon}
         className='ms-PickerItem-content'
-        >
-      </Button>
-    </div >
+        />
+    </div>
+  );
+};
+
+export const SelectedLargeItemDefault: (props: IPeoplePickerItemProps) => JSX.Element = (peoplePickerItemProps: IPeoplePickerItemProps) => {
+  let {
+    item,
+    onRemoveItem,
+    index,
+    isSelected
+  } = peoplePickerItemProps;
+  return (
+    <div
+      className={css('ms-PickerPersona-container', {
```

Comment:
This doesn't seem to follow classname conventions at all.

ms-ClassName-area

You have no class called "PickerPersona" or "PickerItem" which you later reference. please follow conventions.